tests/integration/integration_test_case.py

```diff
@@ -55,7 +55,10 @@ def _set_up_app(self):
 
         configure_logging()
 
-        setting_overrides = {"EQ_ENABLE_HTML_MINIFY": False}
+        setting_overrides = {
+            "EQ_ENABLE_HTML_MINIFY": False,
+            "CONFIRMATION_EMAIL_LIMIT": 2,
```

Comment:
Probably better putting this in the test, as this doesn't make it global to all tests. Being here also hides the fact its being overridden to some extent. We do this in a number of places, for reference look at test_timeout.py

```
    def setUp(self):
        settings.CONFIRMATION_EMAIL_LIMIT = 2
        super().setUp()
```

Reply:
Moved this